platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -5,13 +5,14 @@
 #
 # implementation of new platform api
 #############################################################################
-
+from __future__ import print_function
```

Comment:
@stephenxs why do we need this?

Reply:
by doing so we can take advantage of Python 3 style print function.
In SONiC some of python sources use it some not.